internal/pwru/config.go

```diff
@@ -41,6 +44,30 @@ type FilterCfg struct {
 	IsSet byte
 }
 
+func name2flag(f string) uint8 {
+	switch strings.ToLower(f) {
+	case "fin":
+		return 0x01
+	case "syn":
+		return 0x02
+	case "rst":
+		return 0x04
+	case "psh":
+		return 0x08
+	case "ack":
+		return 0x10
+	case "urg":
+		return 0x20
+	case "ece":
+		return 0x40
+	case "cwr":
+		return 0x80
+	default:
+		log.Fatalf("Invalid tcp flag format")
+		panic("unreachable")
```

Comment:
Nit: this is redundant, as `log.Fatalf()` does `panic()`.

Reply:
yes, it is redundant, but the go compiler doesn't know that. if there panic is not there it will fail to build with "missing return" error.

Reply:
Let's return 0 instead of panic.